libflux/flux-core/src/semantic/types.rs

```diff
@@ -1469,6 +1524,13 @@ mod tests {
         );
     }
     #[test]
+    fn display_type_vector() {
+        assert_eq!(
+            "[int]",
```

Comment:
This is the same as the display for `array`, it should be different. Maybe `{{int}}` or `v[int]`? I am just throwing ideas out there.

Reply:
I have implemented display as v[int]